internal/plan/type.go

```diff
@@ -35,6 +35,8 @@ type Task struct {
 
 // TestPlan represents the entire test plan.
 type TestPlan struct {
+	Identifier   string           `json:"identifier"`
+	Parallelism  int              `json:"parallelism"`
```

Comment:
These fields are already present in the JSON response from the test plan API, but were not declared in this struct.